.github/workflows/ci-backend.yml

```diff
@@ -0,0 +1,38 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI_BACKEND
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the development branch
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      tags:
+        description: 'Test scenario tags'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      
+      # Install dependencies
+      - name: Install dependencies
+        run:  yarn --cwd ./backend/typescript/
+        
+      #Lint Check
+      - name: Run lint for backend
+        working-directory: ./backend/typescript
+        run: yarn lint
```

Comment:
Can you set it so that the lint check fails if there are errors present? Also how bad will it be to also get the code to a point where there are no warnings either?

Reply:
GitHub actions error out automatically when the exit code is 1. [https://github.com/uwblueprint/community-fridge-kw/actions/runs/1273715914](This is a link to community fridge CI workflow) where the job fails on the linting error.